styles/wishlist.css

```diff
@@ -0,0 +1,10 @@
+@import "./card.css";
+
+:root {
+  --card-border-color: #c9c9c9;
+}
+
+#wishlist-cards-wrapper{
+    grid-template-columns: repeat(auto-fit,minmax(25rem, 1fr));
+    grid-gap: 2rem;
```

Comment:
is there any specific reason to use id selector?

Reply:
because i will not repeat those properties for any other element